DD/Views/Partials/_sharedContentPage.cshtml

```diff
@@ -0,0 +1,82 @@
+@using Umbraco.Cms.Web.Common.PublishedModels;
+@using ContentModels = Umbraco.Cms.Web.Common.PublishedModels;
+@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<object>
+@model IHeaderProperties
+@{
+    var richTextStyle = string.Empty;
+    dynamic model;
+    if(Model is ContentModels.JobListingsPage jobListingsPage){
+        model = jobListingsPage;
+    }else if(Model is ContentModels.GeneralContentPage generalContentPage){
+        model = generalContentPage;
+    }else{
+        return;
+    }
+}
+
+<div>
+    @await Html.PartialAsync("homePage/_navigation.cshtml")
+</div>
+<div class="general-content">
+    @foreach (var item in model.ContentPageBlocks)
+    {
+        if (item.Content is ContentModels.LogoItems logoItem)
+        {
+            @await Html.PartialAsync("blocklistItems/_logoItem", logoItem)
+        }
+        if (item.Content is ContentModels.RichTextItems text)
```

Comment:
Try this please if (item.Content is ContentModels.RichTextItems text)
{
    var richTextStyle = $"color:{text.TextColor ?? "inherit"};" +
                        $"font-size:{(text.FontSize != null && text.FontSize != 0 ? text.FontSize : 27)}px;";

    <div class="rich-content-div mt-3" style="@richTextStyle">
        <div class="rich-content-text">
            @Html.Raw(text.RichText)
        </div>
    </div>
}

in order to avoid so many nested if sentences try to use ternary opeators as possible